v2/azure-arm.yaml

```diff
@@ -934,6 +939,23 @@ objectModelConfiguration:
       Factory:
         $export: true
         $supportedFrom: v2.1.0
+  dataprotection: 
+    2023-01-01: 
+      BackupVault: 
+        $export: true 
+        $supportedFrom: v2.2.0 
+      BackupVaults_BackupInstance: 
+        $exportAs: BackupVaultsBackupInstance 
+        $supportedFrom: v2.2.0 
+      BackupVaults_BackupPolicy: 
+        $exportAs: BackupVaultsBackupPolicy 
```

Comment:
Same here.

```suggestion
        $exportAs: BackupPolicy 
```

Reply:
Tried to rectify the same as mentioned, but is hows error. Posting it over Teams to look over.